README.md

```diff
@@ -207,20 +127,7 @@ environment values in your [configuration][config] file:
 ].
 ```
 
-The `dirs` key is a list that indicates where `elvis` should look for the
-files that match `filter`, which will be run through each of the rules
-specified by the `rules` entry, which is a list of items with the following
-structure `{Module, Function, RuleConfig}` or `{Module, Function}` if the rule
-takes no configuration values.
-
-The implementation of a rule is just a function that takes 3 arguments: `elvis`'s
-`config` entry from its [configuration](#configuration); the file to be
-analyzed; and a configuration map specified for the rule. This means you can
-define rules of your own as long as the functions that implement them respect
-this arity.
-
-There's currently no default configuration for `elvis`, but in the meantime
-you can take the one in `config/elvis.config` as a starting point.
+The `config` and `output_format` are explained in the [`elvis_core`](https://github.com/inaka/elvis_core).
```

Comment:
explained in the -> explained in | explained in the [`elvis_core`](https://github.com/inaka/elvis_core) repository.